{"name": "test_login_success", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to have text 'test1111'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".username\")", "trace": "self = <test_login.TestLogin object at 0x0000017DE643E3F0>, page = <Page url='https://hocvancokimngan.com/login/'>\n\n    @allure.story(\"Login thành công\")\n    def test_login_success(self, page):\n        lp = LoginPage(page)\n>       lp.goto().login(settings.USER_EMAIL, settings.USER_PASSWORD).assert_logged_in(\"test1111\")\n\ntests\\web\\test_login.py:14: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <pages.login_page.LoginPage object at 0x0000017DE7E60CE0>, username = 'test1111'\n\n    def assert_logged_in(self, username):\n        with allure.step(\"Xác minh đã đăng nhập thành công\"):\n>           expect(self.page.locator(self.lbl_username)).to_have_text(username)\nE           AssertionError: Locator expected to have text 'test1111'\nE           Actual value: <element(s) not found> \nE           Call log:\nE           LocatorAssertions.to_have_text with timeout 5000ms\nE             - waiting for locator(\".username\")\n\npages\\login_page.py:39: AssertionError"}, "steps": [{"name": "Mở trang chủ", "status": "passed", "start": 1755784132744, "stop": 1755784134220}, {"name": "Đi tới trang Sign in", "status": "passed", "start": 1755784134220, "stop": 1755784135085}, {"name": "Nhập thông tin đăng nhập", "status": "passed", "start": 1755784135085, "stop": 1755784135174}, {"name": "Đăng nhập", "status": "passed", "start": 1755784135174, "stop": 1755784136655}, {"name": "Xác minh đã đăng nhập thành công", "status": "failed", "statusDetails": {"message": "AssertionError: Locator expected to have text 'test1111'\nActual value: <element(s) not found> \nCall log:\nLocatorAssertions.to_have_text with timeout 5000ms\n  - waiting for locator(\".username\")\n\n", "trace": "  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\pages\\login_page.py\", line 39, in assert_logged_in\n    expect(self.page.locator(self.lbl_username)).to_have_text(username)\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\sync_api\\_generated.py\", line 19324, in to_have_text\n    self._sync(\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_sync_base.py\", line 115, in _sync\n    return task.result()\n           ^^^^^^^^^^^^^\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 481, in to_have_text\n    await self._expect_impl(\n  File \"C:\\Users\\thanh\\PycharmProjects\\auto_python_playwright_web\\.venv\\Lib\\site-packages\\playwright\\_impl\\_assertions.py\", line 74, in _expect_impl\n    raise AssertionError(\n"}, "start": 1755784136655, "stop": 1755784141672}], "start": 1755784132744, "stop": 1755784141677, "uuid": "71476a58-47f5-43ce-8f20-ae136e9215d4", "historyId": "875a81a05b6adf5de97d29a15a01413b", "testCaseId": "875a81a05b6adf5de97d29a15a01413b", "fullName": "tests.web.test_login.TestLogin#test_login_success", "labels": [{"name": "severity", "value": "critical"}, {"name": "story", "value": "Login thành công"}, {"name": "feature", "value": "Authentication"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "tests.web"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "Thanh-PC"}, {"name": "thread", "value": "7580-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.web.test_login"}]}